jobs/config.json

```diff
@@ -195,7 +205,9 @@
     "args": [
       "--env-file=platforms/gce.env", 
       "--env-file=jobs/ci-kubernetes-e2e-gce-1.5-1.6-cvm-kubectl-skew.env", 
-      "--timeout=120m"
+      "--timeout=120m", 
+      "--extract=ci/latest-1.6", 
+      "--extract=ci/latest-1.5"
```

Comment:
It is right, the old values were:

```
JENKINS_PUBLISHED_SKEW_VERSION=ci/latest-1.6		
JENKINS_PUBLISHED_VERSION=ci/latest-1.5
```

And if you look at the old `e2e-runner.sh logic` you'll notice that the `JENKINS_PUBLISHED_SKEW_VERSION --extract` gets added first, same as here.